### PR TITLE
Carbon Tax, Council Tax, taxable and means-test-included UBI

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
# New features: Carbon Tax, Council Tax, taxable and means-test-included UBI

Adds the above mentioned.

### CODE REVIEW

File: policyengine/countries/uk/reform_parameters.yaml [hunk 1]
```diff
@@ -76,6 +76,24 @@ reforms:
           description: Age at which a person receives the working-age adult UBI payment.
           max: 65
           summary: Set the working-age UBI age threshold to @
+    taxable:
+      values: 
+        2021-01-01: false
+      metadata:
+        policyengine:
+          title: Taxable UBI
```
Comment (MaxGhenis) [inline policyengine/countries/uk/reform_parameters.yaml:84]:
```suggestion
          title: UBI taxability
```

File: policyengine/countries/uk/reform_parameters.yaml [hunk 2]
```diff
@@ -76,6 +76,24 @@ reforms:
...
+          short_name: taxable_UBI
+          type: bool
+          description: Whether the UBI is included in taxable income for Income Tax.
+    in_means_tests:
+      values: 
+        2021-01-01: false
+      metadata:
+        policyengine:
+          title: Included in means tests
```
Comment (MaxGhenis) [inline policyengine/countries/uk/reform_parameters.yaml:93]:
```suggestion
          title: UBI included in means tests
```
For consistency. Could also be just `Taxability` / `Included in means tests`.

**Review by MaxGhenis — APPROVED** [review]
This will be amazing...but, yikes this creates a lot of code duplication. Refactoring any of these formulas (and I see a fair amount of opportunity for it) will now require parallel PRs across repos. OK for now but would love to explore alternatives for the future.

File: policyengine/countries/uk/reform_parameters.yaml (hunk 2 above)
Comment (nikhilwoodruff) [reply policyengine/countries/uk/reform_parameters.yaml]: Thanks, added

File: policyengine/countries/uk/reform_parameters.yaml (hunk 1 above)
Comment (nikhilwoodruff) [reply policyengine/countries/uk/reform_parameters.yaml]: Added

**Pull request merged**